### PR TITLE
adds link to feedback form

### PR DESCRIPTION
**Pull request by akash1810**
screen shot 2017-01-12 at 09 22 43

### CODE REVIEW

**Comment by mr-mr** [comment]
👍
Nice. Brings up a question about the green being used on the edit screen buttons. Should we move to the yellow as the primary colour?

File: public/video-ui/src/components/Header.js [hunk 1]
```diff
@@ -17,11 +17,18 @@ export default class Header extends React.Component {
           </div>
 
           <div className="topbar__container">
+            <nav className="topbar__nav topbar__feedback">
+              <a className="topbar__nav-link"
+                 target="_blank"
+                 href="https://goo.gl/forms/0KoeGOW64584Bydm2">
+                Give us feedback?
```
Comment (mr-mr) [inline public/video-ui/src/components/Header.js]: Can we just say Give feedback, rather than ask a question.

**Comment by akash1810** [comment]
@mr-mr yellow was chosen as the logo is yellow and they both sit in the top bar. Not too fussed on colour choice, but have changed to `brandColor` (green); can always change later.

File: public/video-ui/src/components/Header.js (hunk 1 above)
Comment (akash1810) [inline public/video-ui/src/components/Header.js]: Done.

**Comment by mr-mr** [comment]
I should have been more explicit! I like the yellow, it works better on the grey!
I'll look at the colours as a whole this morning.

**Pull request merged**